ratatool-diffy/src/test/scala/com/spotify/ratatool/diffy/BigDiffyTest.scala

```diff
@@ -120,6 +120,32 @@ class BigDiffyTest extends PipelineSpec {
     }
   }
 
+  it should "work with NaNs" in {
```

Comment:
can we also test for when `ignoreNan` is false (the default)?

Reply:
nit: more descriptive name(s) for the tests would also be good

Reply:
Cool, good call.  Added another test, but couldn't verify min/max since how Algebird computes min/maxes when NaN values are present will result in flaky tests - added a comment about this in the new test in 3c81e32